packages/camera/camera/pubspec.yaml

```
@@ -2,7 +2,7 @@ name: camera
 description: A Flutter plugin for getting information about and controlling the
   camera on Android and iOS. Supports previewing the camera feed, capturing images, capturing video,
   and streaming image buffers to dart.
-version: 0.6.3
+version: 0.6.4
```

Change to indicate a patch update:
```suggestion
version: 0.6.3+1
```